kubernetes/go-gitops/waypoint.hcl

```
@@ -0,0 +1,63 @@
+project = "go-gitops-0"
+
+variable "image" {
+  default     = "your.own.container.registry.example.com/go"
+  type        = string
+  description = "Image name for the built image in the Docker registry."
+}
+
+variable "tag" {
+  default     = "latest"
+  type        = string
+  description = "Image tag for the image"
+}
+
+// set your registry username through the UI or on the CLI!
+variable "registry_username" {
+  default     = ""
+  type        = string
+  description = "username for container registry"
+}
+
+// set your registry password through the UI or on the CLI!
+variable "registry_password" {
+  default     = ""
+  type        = string
+  description = "password for registry" // DO NOT COMMIT YOUR PASSWORD TO GIT
+}
```

```suggestion
variable "registry_password" {
  default     = ""
  type        = string
  description = "password for registry" // DO NOT COMMIT YOUR PASSWORD TO GIT
  env = ["REGISTRY_PASSWORD"]
}
```

And maybe add `env = ["REGISTRY_USERNAME"]` as well. Later we could go even further and instruct how to set these envs in a runner profile(?)

---

Hm, I don't like suggesting people put container registry passwords in their environment. If you wanted to, I think you'd have to set it in a runner profile yeah.

I think the easiest way is to just tell people to set these input variables through the project config in the UI or on the command line if running Waypoint locally (via `waypoint up -var registry_password=blah`)